packages/Webkul/Velocity/src/Resources/views/shop/home/advertisements/advertisement-four.blade.php

```diff
@@ -37,7 +37,7 @@ class="{{ $isLazyLoad ? 'lazyload' : '' }}"
                     @endif
                 </div>
 
-                <div class="col-lg-4 col-12 advertisement-container-block offers-ct-panel">
+                <div class="advertisement-container-block adv2">
```

Comment:
Use a meaningful name like the first one is `advertisement-container-block` then the remaining classes should follow the convention like `advertisement-1` or may be try to handle without numbering like `advertisement-container-block advertisement-small advertisement-large advertisement-medium`  whatever is suitable.